docs/code/deployments-kubernetes/deployment-opa.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     spec:
       containers:
       - name: opa
-        image: openpolicyagent/opa:0.10.5
+        image: openpolicyagent/opa:0.10.6
```

Comment:
@lucperkins does this have to be manually maintained? If we can't template the YAML files we can just move the content into markdown.

Reply:
At the moment this needs to be manually maintained but I'm going to explore if there's a way to templatize code snippets imported using Hugo's [`readFile`](https://gohugo.io/functions/readfile/) function (which is used by that `code` import function). If not, I'll just copy and paste into the Markdown. I can include that in a follow-up PR next week. I do 100% agree that that should not be manually maintained.